### PR TITLE
FR: add object latest with values of last charging session

### PR DESCRIPTION
**Pull request by grueni**
The response JSON from BMW is not very handy for the import if the charging sessions into a database.
To improve the import of the charging session of the charging sessionss into a database I copied the last charging session in the object 'latest'. 
In the object I added the fields unit, timestamp and the original id.
With this object the import in a database is very easy.
Please see also in here: #6

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -390,8 +390,26 @@ class Bmw extends utils.Adapter {
             },
             native: {},
           });
+          await this.extractKeys(this, vin + element.path + dateFormatted, data);
```
Comment (TA2k) [inline main.js:393]: This will lead to infinite datapoints if you add every day a new channel

Comment (grueni) [reply main.js:393]: I have to admit that in nmpjs and iobroker I am a newbie.
This is the original line of your source code prepended with an 'await'.
Is this the consequence of the await?

-

Comment (TA2k) [reply main.js:393]: No its not a nodejs or iobroker issue.
It was a misreading by me. I already creating for every session a datapoint which fills up the iobroker. But so far no complaints

**Comment by TA2k** [comment]
Merged and improved

**Pull request merged**